data/default/health/field-app-configuration.json

```diff
@@ -18,6 +18,20 @@
           "value": "en_MZ"
         }
       ],
+      "GENDER_OPTIONS_POPULATOR":[
+           {
+              code: "MALE",
+              name: "MALE",
+            },
+            {
+              code: "FEMALE",
+              name: "FEMALE",
+            },
+            {
+              code: "TRANSGENDER",
```

Comment:
The enum on the backend has possible values as MALE/FEMALE/OTHERS
https://editor.swagger.io/?url=https://raw.githubusercontent.com/digit-egov/health-api-specs/main/contracts/registries/individual.yml